src/lib/coerce.js

```diff
@@ -44,6 +44,20 @@ exports.valObjects = {
             if(opts.coerceNumber) v = +v;
             if(opts.values.indexOf(v) === -1) propOut.set(dflt);
             else propOut.set(v);
+        },
+        validateFunction: function(v, opts) {
```

Comment:
This thing in only called during `Lib.validate`. We need this so that make enumerated attributes like `xaxis.anchor` that declare [regex](https://github.com/plotly/plotly.js/blob/9779c9608ff8554a074285eac67892dc100af27b/src/plots/cartesian/layout_attributes.js#L512-L513) work with `Lib.validate`.